appcues/src/main/java/com/appcues/analytics/ExperienceLifecycleTracker.kt

```diff
@@ -92,20 +81,20 @@ internal class ExperienceLifecycleTracker(
         analyticsTracker.track(event.name, properties, interactive = false, isInternal = true)
     }
 
-    private fun State.shouldTrack(): Boolean =
+    private fun State.shouldNotTrackUnpublished(): Boolean =
```

Comment:
this feels a little less readable to me, personally. Could keep `shouldTrack()` and check the `.not()` case above on line 35?

Reply:
Nitpick, but following changes here, I think this was updated to `shouldTrackUnpublished`? I was thinking just keep `shouldTrack()` as the implementation details of what constitutes a valid item to track should be contained within (in this case, checks for published content).